src/rta/supernode.cpp

```diff
@@ -167,6 +167,9 @@ Supernode *Supernode::createFromViewOnlyWallet(const string &path, const string
 
     result = new Supernode(testnet);
     result->m_wallet->generate(path, password, wallet_addr, viewkey);
+    // we don't know in advance when announced supernode wallet has been created, supposing it have been creating at
+    // any time since beginning, so refreshing it from block height 0
+    result->m_wallet->set_refresh_from_block_height(0);
```

Comment:
This isn't needed here, and won't fix the issue: a view-only wallet is always created with a refresh-from-block of 0.  The issue instead affects the auto-generated stake wallet.  I think this needs to move into the constructor instead, just after line 51 (the `m_wallet->generate(...)` call).

Reply:
True, I misunderstood the issue. fixed now